client-src/elements/chromedash-stack-rank.js

```diff
@@ -178,10 +187,19 @@ class ChromedashStackRank extends LitElement {
     this.viewList = sortBy_(this.sortType, this.sortReverse, newViewList);
   }
 
+  handleChangeHideObsolete(e) {
+    this.shouldHideObsolete = e.target.checked;
+  }
+
   renderSubHeader() {
     return html`
       <div id="subheader">
-        <p class="title-text">Showing <span>${this.viewList.length}</span> properties</p>
+        <p class="title-text">Showing <span>${this.viewList.length - (
+          this.shouldHideObsolete ? this.obsoleteCount : 0
+        )}</span> properties</p>
+        <sl-checkbox ?checked=${this.shouldHideObsolete} @input=${this.handleChangeHideObsolete} >
```

Comment:
Why `@input` rather than `@sl-change`?

Reply:
I found `@input` here:

https://github.com/GoogleChrome/chromium-dashboard/blob/ae58989d1e1e16e42e18501df7776466dc1c6db8/client-src/elements/chromedash-settings-page.js#L102

I was wondering why `@input` works. I checked
https://github.com/shoelace-style/shoelace/blob/next/src/components/checkbox/checkbox.ts#L106 
only emits events starts with `sl-`.

`@sl-change` is documented but for now I leave it as it is.